babel.config.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+	presets: [
+		[
+			'@babel/preset-env',
+			{
+				exclude: [
+					'transform-async-to-generator',
+					'proposal-async-generator-functions',
+					'transform-regenerator'
+				],
+				targets: {
+					chrome: '58',
+					ie: '11',
+				},
+			},
+		],
+	],
+	plugins: [
+		'babel-plugin-transform-async-to-promises',
```

Comment:
why not fast-async?

Reply:
Because it also relies on `nodent` and relying on that again would lead to the same JSX error message.

One could explore to make sure JSX is transformed before passing it to nodent and making absolutely sure that nothing else loads a newer `acorn` version, but that seems fragile to me.

Reply:
Well, we don't need the jsx plugin if we do that first in another transform